fetcher.sh

```diff
@@ -64,6 +65,39 @@ if [ "$kernel" = "Linux" ]; then
 	cpu="$(awk -F': ' '/model name\t: /{print $2;exit} ' "/proc/cpuinfo")"
 	ram="$(awk '/[^0-9]* / {print $2" "$3;exit} ' "/proc/meminfo")"
 
+        # gpu
+        if
+          [ "$(lspci | grep -i vga | grep -i nvidia | awk '{print $5}')" ]
+        then
+          gpu="$(lspci | grep -iE 'vga|display|3d' | grep -iE nvidia | awk '{print $5 " " $8 " " $9 " " $10 " " $11}'| sed -e 's/\[\|\]//g')"
+        elif
+          [ "$(lspci | grep -iE 'vga|display|3d' | grep -i amd)" ]
+        then
+          gpu="$(lspci | grep -iE 'vga|display|3d' | grep -iE amd | awk '{print $5 " " $9 " " $11 " " $12}' | sed -e 's/\[\|\]//g')"
+        elif
+          [ "$(lspci | grep -iE 'vga|display|3d' | grep -i radeon)" ]
+        then
+           gpu="$(lspci | grep -iE 'vga|display|3d' | grep -iE radeon | awk '{print $5 " " $9 " " $11 " " $12}' | sed -e 's/\[\|\]//g')"
+        elif
+          [ "$(lspci | grep -iE 'vga|display|3d' | grep -i rx)" ]
+        then
+          gpu="$(lspci | grep -iE 'vga|display|3d' | grep -iE rx | awk '{print $5 " " $9 " " $11 " " $12}' | sed -e 's/\[\|\]//g')"
+        elif
+          [ "$(lspci | grep -iE 'vga|display|3d' | grep -i xt)" ]
+        then
+          gpu="$(lspci | grep -iE 'vga|display|3d' | grep -iE xt | awk '{print $5 " " $9 " " $11 " " $12}' | sed -e 's/\[\|\]//g')"
+        elif
+          [ "$(lspci | grep -iE 'vga|display|3d' | grep -i vega)" ]
+        then
+          gpu="$(lspci | grep -iE 'vga|display|3d' | grep -iE vega | awk '{print $5 " " $9 " " $11 " " $12}' | sed -e 's/\[\|\]//g')"
+        elif
+          [ "$(lspci | grep -i vga | grep -i intel | awk '{print $5}')" ]
+        then
+          gpu="$(lspci | grep -E 'VGA|Display' | grep -iE intel | awk '{print $5 " " $8 " " $9 " " $10 " " $11}' | cut -d'(' -f1)"
+        else
+          gpu=" "
+        fi
+
```

Comment:
i.... have nothing to say.

I'll see you in 4 hours after i claw my eyes out and drink some bleach. Peace my dude!